hack/generate-knative.sh

```diff
@@ -82,11 +82,15 @@ if grep -qw "injection" <<<"${GENS}"; then
   # Clear old injection
   rm -rf ${OUTPUT_PKG}
 
+  MODULE_NAME=$(go list -m)
```

Comment:
This is just `knative.dev/pkg` it seems. Do we need this?

Reply:
this script is used in other repos to generate their injection code - so it's not always knative.dev/pkg

Reply:
Ah correct ok